icon_validator/rules/plugin_validators/cloud_ready_validator.py

```diff
@@ -17,10 +17,11 @@ def validate_enable_cache_in_plugin_spec(plugin_spec: dict):
 
     @staticmethod
     def validate_python_version_in_dockerfile(dockerfile: str):
-        if not re.compile(r"FROM rapid7/insightconnect-python-3-38(-slim)?-plugin:(([0-9][0-9]+)|[4-9])")\
-                .match(dockerfile):
+        docker_str = dockerfile.replace(" --platform=linux/amd64 ", " ")  # if user has specified '--platform' remove
+        if not re.compile(r"FROM rapid7/insightconnect-python-3(-slim)?-plugin:(([0-9][0-9]+)|[4-9])")\
+                .match(docker_str):
```

Comment:
in the case we have 

`FROM --platform=linux/amd64 rapid7/insightconnect-python-3-plugin:latest`

this will fail on this regex as its only looking for a number and not also supporting latest

Reply:
Updated to allow for this and added unit test to cover it